src/platform/serviceRegistry.node.ts

```diff
@@ -5,9 +5,10 @@
 import { IServiceManager } from '../platform/ioc/types';
 import { CommandRegistry } from './commands/commandRegistry.node';
 import { registerTypes as registerApiTypes } from './api/serviceRegistry.node';
-import { registerTypes as commonRegisterTypes } from './common/serviceRegistry.node';
-import { registerTypes as commonRegisterTerminalTypes } from './terminals/serviceRegistry.node';
-import { registerTypes as activationRegisterTypes } from './activation/serviceRegistry.node';
+import { registerTypes as registerCommonTypes } from './common/serviceRegistry.node';
```

Comment:
This was review feedback that @IanMatthewHuff gave. I agree, names were inconsistent.